db-manager/tasks/genericTask.js

```diff
@@ -162,24 +162,68 @@ async function genericTask(taskInput, db, seedId, callback) {
         if (targetTask.type === "xchain") {
           console.log("--- xchain task");
           const linkedWallets = validUser.linkedWallets;
-          if (linkedWallets.length === 0 || linkedWallets == null) {
+          if (linkedWallets.length === 0) {
             console.log(
               `--- User ${validUser._id} does not have any linked wallet`,
             );
             continue;
           }
-          // execute the logic per each xchain wallet
-          for (const xChainWallet of linkedWallets) {
-            if (xChainWallet.type === "evm") {
-              for (const chain of chains.evm) {
-                const userWallet = `${chain}/${xChainWallet.address}`;
+
+          if (targetTask.seedId === "MINTING_BNUSD_SUI" || targetTask.seedId === "DEPOSIT_NATIVE_SUI_COLLATERAL") {
```

Comment:
the tasks labels are being imported as TASK_LABELS at the top of the file, to avoid having string literals in the code we can change `targetTask.seedId === "MINTING_BNUSD_SUI` to `targetTask.seedId === TASK_LABELS.mintingBnusdSui`, and same for `DEPOSIT_NATIVE_SUI_COLLATERAL`